src/components/ButtonIcon/styled/button.js

```diff
@@ -150,7 +172,7 @@ const StyledButton = styled.button`
     ${props =>
         props.variant === 'border-filled' &&
         `
-            background-color: ${COLOR_WHITE};
+            background-color: transparent;
```

Comment:
This variant needs to have background-color since with this change then this variant becomes the same as the `border` variant